### ISSUE TITLE
[Docs page] Community 

### ISSUE THREAD

**Issue by schersh** [hashicorp/terraform-cdk#969]
**Description**
This page should point users to places where they can contribute, submit issues, and discuss CDKTF with other users. It should explain:
- How to contribute
- Questions go on discuss: https://discuss.hashicorp.com/c/terraform-core/cdk-for-terraform/47
- Bugs and feature requests filed as tickets https://github.com/hashicorp/terraform-cdk/issues

Content should go in: [/website/docs/cdktf/community.html.md](https://github.com/hashicorp/terraform-cdk/blob/terraform-io-docs/website/docs/cdktf/community.html.md)

**Engineering support** is needed for:
- Reviewing existing content (already on page) for accuracy
- Adding additional content that is relevant and helpful to users

---

Add info in readme about forums, tickets, etc. - DONE

**Comment by danieldreier** [comment]
@laurapacilio I've made a PR with a revision and some small changes

**Comment by laurapacilio** [comment]
Perfect - I just reviewed it with a couple of places where I suggest rewording, but it looks great!

**Comment by schersh** [comment]
merged to `stable-website`
